### ISSUE TITLE
Quick Start v2 VoiceOver: toast messages are not accessible

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#11496]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. If you already have a site with Quick Start setup, go to that site, double tap on "Customize Your Site", and skip to step 7.
1. Log out of the WPiOS app.
1. Go to WPiOS and double tap the "Sign up for WordPress.com" button.
1. Follow the prompts until you reach "Want a little help getting started?"
1. Double tap the "Yes, Help Me" button.
1. Double tap the "x" button.
1. Try to tap "Not now" or "Yes, show me" in the toast message that appears.
1. Double tap the toast message that says "Tap Your Site Icon to upload a new one."

Result: all the references to “Tap” should be read as “Double tap” by VoiceOver. Icons are not read by VoiceOver. Things like “Tap + to do something” is read as “Tap to do something” Changing its meaning completely. ([57s](https://cloudup.com/cg1o0H9PgvC))

<img src="https://user-images.githubusercontent.com/1119271/56328966-9f4cfc00-613e-11e9-9c0f-d5b9d3686fcf.png" width="270" alt="wpios-voiceover-toast-message">
<p style="clear:both;"><sup>Tested with WP Internal 12.2.0.20190412 on iPhone 6S iOS 12.2.</sup></p>

<p style="clear:both;"><sub>(internal reference: p77Llu-bQ6-p2 h/t @etoledom)</sub></p>

**Comment by stevebaranski** [comment]
Given a description, specifying a `accessibilityLabel` is straightforward. I took some time to evaluate this issue, and I'd like to share my findings with regard to _Quick Start_.

- We'll likely need to amend the [`Waypoint` tuple](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/Blog/QuickStartTours.swift#L4), and specify the value (i.e., the accessible description to be set) to use in each `QuickStartTour` implementor.
- That description will need to be injected when a "step notice" is created [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift#L327-L334). It might be useful to include this additional attribute as part of the [`NoticeStyle`](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/System/Notices/NoticeStyle.swift#L3), where the `attributedMessage` is currently specified. Otherwise the `NoticeStore` [initializer](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Stores/NoticeStore.swift#L49-L67) will need to be modified to accept such a description.
- If an accessibility description is specified (i.e., not `nil`), the `NoticeView` will need to set the `accessibilityLabel`, probably [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/System/Notices/NoticeView.swift#L225-L236).

**Comment by shiki** [comment]
Removed “Quick Start” from the name since this affects all places where we use this UI element — Notice.

**Comment by shiki** [comment]
#12873 Adds announcements when Notices are displayed so they are not invisible for VoiceOver users. 

> Icons are not read by VoiceOver. Things like “Tap + to do something” is read as “Tap to do something” Changing its meaning completely

I could not reproduce this. But maybe I fixed it in #12873. 😄 

I updated the description ☝️ to clarify what is left with this issue.
